### PR TITLE
Bools  and Ghost-Gobble Arcade Game Typos and Revisions

### PR DESCRIPTION
**Pull request by BethanyG**
Went through the docs for the `bools` concept & `Ghost Gobble Arcade Game` concept exercise, editing for the following:

 - [x] Spelling
 - [x] Language
 - [x] Readability & concept coverage
 - [x] Accurate code examples, syntax, and PEP-8

Also added more formal docstrings to the stub file.

### CODE REVIEW

File: concepts/bools/about.md [hunk 1]
```diff
@@ -1,21 +1,25 @@
 # About
 
-Python represents True and False values with the [bool][bool] type. There are only two boolean values: `True` and `False`. These values can be assigned to a variable and combined with the [boolean operators][boolean-operators] (`and`, `or`, `not`):
+Python represents True and False values with the [bool][bool] type.
+ There are only two boolean values in this type: `True` and `False`.
+  These values can be assigned to a variable and combined with the [boolean operators][boolean-operators] (`and`, `or`, `not`):
```
Comment (kotp) [inline concepts/bools/about.md:5]: I am not positive, but I believe "Boolean" is meant to be capitalized, due to honoring the name of the inventor.  However, I am pretty sure I am not mistaken.

File: concepts/bools/about.md [hunk 2]
```diff
@@ -27,56 +31,68 @@ True
 
 All `boolean operators` are considered lower precedence than Pythons [`comparison operators`][comparisons], such as `==`, `>`, `<`, `is` and `is not`.
 
+
 ## Type Coercion and Truthiness
 
-The bool function ([`bool()`][bool-function]) converts any object to a Boolean value. By default all objects return `True` unless defined to return `False`.
+The bool function ([`bool()`][bool-function]) converts any object to a Boolean value.
```
Comment (kotp) [inline concepts/bools/about.md:37]:
```suggestion
The `bool` function ([`bool()`][bool-function]) converts any object to a Boolean value.
```

Because that is a function name, and code, so code "markuped".  Also, reinforcing my prior mention of capitalization of the word "Boolean", it is done here, and so the prior uncapitalized name seems out of place even more.

File: concepts/bools/about.md [hunk 3]
```diff
@@ -27,56 +31,68 @@ True
...
 False
 
 >>>bool({"Pig" : 1, "Cow": 3})
 True
+
 >>>bool({})
 False
 ```
 
 When a object is used in a _boolean context_, it is evaluated transparently as _truthy_ or _falsey_ using `bool()`:
```
Comment (kotp) [inline concepts/bools/about.md:70]:
```suggestion
When an object is used in a _boolean context_, it is evaluated transparently as _truthy_ or _falsey_ using `bool()`:
```

File: concepts/bools/introduction.md [hunk 4]
```diff
@@ -1,21 +1,23 @@
...
+  These values can be bound to a variable:
 
 ```python
-true_variable = True
-false_variable = False
+>>> true_variable = True
+>>> false_variable = False
 ```
 
 We can evaluate boolean expressions using the `and`, `or`, and `not` operators.
```
Comment (kotp) [inline concepts/bools/introduction.md:12]:
```suggestion
We can evaluate Boolean expressions using the `and`, `or`, and `not` operators.
```

This was not part of the change, and there may be some references to "boolean" vs "Boolean" that I am not seeing…

File: exercises/concept/ghost-gobble-arcade-game/.docs/hints.md [hunk 5]
```diff
@@ -2,23 +2,24 @@
 
 ## General
 
-- Don't worry about how the arguments are derived, just focus on combining the arguments to return the intended result.
+- Don't worry about how the arguments are _derived_, focus on combining the arguments to return the intended result.
 
 ## 1. Define if Pac-Man can eat a ghost
 
-- You can use the [boolean][boolean] [operators][boolean-operators] to combine the arguments for a result.
+- You can use the [boolean][boolean] [operators][boolean-operators] to combine arguments for a result.
```
Comment (kotp) [inline exercises/concept/ghost-gobble-arcade-game/.docs/hints.md:9]:
```suggestion
- You can use the [Boolean][boolean] [operators][boolean-operators] to combine arguments for a result.
```

File: exercises/concept/ghost-gobble-arcade-game/.docs/hints.md [hunk 6]
```diff
@@ -2,23 +2,24 @@
...
 
 ## 2. Define if Pac-Man scores
 
-- You can use the [boolean][boolean] [operator][boolean-operators] to combine the arguments for a result.
+- You can use the [boolean][boolean] [operator][boolean-operators] to combine arguments for a result.
```
Comment (kotp) [inline exercises/concept/ghost-gobble-arcade-game/.docs/hints.md:13]:
```suggestion
- You can use the [Boolean][boolean] [operator][boolean-operators] to combine arguments for a result.
```

File: exercises/concept/ghost-gobble-arcade-game/.docs/hints.md [hunk 7]
```diff
@@ -2,23 +2,24 @@
...
 
 ## 3. Define if Pac-Man loses
 
-- You can use the [boolean][boolean] [operator][boolean-operators] to combine the arguments for a result.
+- You can use the [boolean][boolean] [operator][boolean-operators] to combine arguments for a result.
```
Comment (kotp) [inline exercises/concept/ghost-gobble-arcade-game/.docs/hints.md:17]:
```suggestion
- You can use the [boolean][Boolean] [operator][boolean-operators] to combine arguments for a result.
```

File: exercises/concept/ghost-gobble-arcade-game/.docs/hints.md [hunk 8]
```diff
@@ -2,23 +2,24 @@
...
 
 ## 4. Define if Pac-Man wins
 
-- You can use the [boolean][boolean] [operator][boolean-operators] to combine the arguments for a result.
+- You can use the [boolean][boolean] [operator][boolean-operators] to combine arguments for a result.
```
Comment (kotp) [inline exercises/concept/ghost-gobble-arcade-game/.docs/hints.md:21]:
```suggestion
- You can use the [Boolean][boolean] [operator][boolean-operators] to combine arguments for a result.
```

File: exercises/concept/ghost-gobble-arcade-game/.docs/instructions.md [hunk 9]
```diff
@@ -1,43 +1,53 @@
 # Instructions
 
-In this exercise, you need to implement some rules from [Pac-Man, the classic arcade-game](https://en.wikipedia.org/wiki/Pac-Man).
+In this exercise, you need to implement some rules from [Pac-Man][Pac-Man], the classic 1980s-era arcade-game.
 
 You have four rules to implement, all related to the game states.
 
-> Don't worry about how the arguments are derived, just focus on combining the arguments to return the intended result.
+> _Don't worry about how the arguments are derived, just focus on combining the arguments to return the intended result._
```
Comment (kotp) [inline exercises/concept/ghost-gobble-arcade-game/.docs/instructions.md:7]:
```suggestion
> _Do not worry about how the arguments are derived, just focus on combining the arguments to return the intended result._
```

File: exercises/concept/ghost-gobble-arcade-game/.docs/instructions.md [hunk 10]
```diff
@@ -1,43 +1,53 @@
...
 
 ## 1. Define if Pac-Man eats a ghost
 
-Define the `eat_ghost()` function that takes two parameters (_if Pac-Man has a power pellet active_ and _if Pac-Man is touching a ghost_) and returns a boolean value if Pac-Man is able to eat the ghost. The function should return `True` only if Pac-Man has a power pellet active and is touching a ghost.
+Define the `eat_ghost()` function that takes two parameters (_if Pac-Man has a power pellet active_ and _if Pac-Man is touching a ghost_) and returns a boolean value if Pac-Man is able to eat the ghost.
```
Comment (kotp) [inline exercises/concept/ghost-gobble-arcade-game/.docs/instructions.md:11]:
```suggestion
Define the `eat_ghost()` function that takes two parameters (_if Pac-Man has a power pellet active_ and _if Pac-Man is touching a ghost_) and returns a Boolean value if Pac-Man is able to eat the ghost.
```

File: exercises/concept/ghost-gobble-arcade-game/.docs/instructions.md [hunk 11]
```diff
@@ -1,43 +1,53 @@
...
-# => False
+>>> eat_ghost(False, True)
+...
+False
 ```
 
 ## 2. Define if Pac-Man scores
 
-Define the `score()` function that takes two parameters (_if Pac-Man is touching a power pellet_ and _if Pac-Man is touching a dot_) and returns a boolean value if Pac-Man scored. The function should return `True` if Pac-Man is touching a power pellet or a dot.
+Define the `score()` function that takes two parameters (_if Pac-Man is touching a power pellet_ and _if Pac-Man is touching a dot_) and returns a boolean value if Pac-Man scored.
```
Comment (kotp) [inline exercises/concept/ghost-gobble-arcade-game/.docs/instructions.md:22]:
```suggestion
Define the `score()` function that takes two parameters (_if Pac-Man is touching a power pellet_ and _if Pac-Man is touching a dot_) and returns a Boolean value if Pac-Man scored.
```

File: exercises/concept/ghost-gobble-arcade-game/.docs/instructions.md [hunk 12]
```diff
@@ -1,43 +1,53 @@
...
-# => True
+>>> score(True, True)
+...
+True
 ```
 
 ## 3. Define if Pac-Man loses
 
-Define the `lose()` function that takes two parameters (_if Pac-Man has a power pellet active_ and _if Pac-Man is touching a ghost_) and returns a boolean value if Pac-Man loses. The function should return `True` if Pac-Man is touching a ghost and does not have a power pellet active.
+Define the `lose()` function that takes two parameters (_if Pac-Man has a power pellet active_ and _if Pac-Man is touching a ghost_) and returns a boolean value if Pac-Man loses.
```
Comment (kotp) [inline exercises/concept/ghost-gobble-arcade-game/.docs/instructions.md:33]:
```suggestion
Define the `lose()` function that takes two parameters (_if Pac-Man has a power pellet active_ and _if Pac-Man is touching a ghost_) and returns a Boolean value if Pac-Man loses.
```

File: exercises/concept/ghost-gobble-arcade-game/.docs/instructions.md [hunk 13]
```diff
@@ -1,43 +1,53 @@
...
-# => True
+>>> lose(False, True)
+...
+True
 ```
 
 ## 4. Define if Pac-Man wins
 
-Define the `win()` function that takes three parameters (_if Pac-Man has eaten all of the dots_, _if Pac-Man has a power pellet active_, and _if Pac-Man is touching a ghost_) and returns a boolean value if Pac-Man wins. The function should return `True` if Pac-Man has eaten all of the dots and has not lost based on the parameters defined in part 3.
+Define the `win()` function that takes three parameters (_if Pac-Man has eaten all of the dots_, _if Pac-Man has a power pellet active_, and _if Pac-Man is touching a ghost_) and returns a boolean value if Pac-Man wins.
```
Comment (kotp) [inline exercises/concept/ghost-gobble-arcade-game/.docs/instructions.md:44]:
```suggestion
Define the `win()` function that takes three parameters (_if Pac-Man has eaten all of the dots_, _if Pac-Man has a power pellet active_, and _if Pac-Man is touching a ghost_) and returns a Boolean value if Pac-Man wins.
```

File: exercises/concept/ghost-gobble-arcade-game/.docs/introduction.md [hunk 14]
```diff
@@ -1,21 +1,23 @@
...
 
 ```python
-true_variable = True
-false_variable = False
+>>> true_variable = True
+>>> false_variable = False
 ```
 
-We can evaluate boolean expressions using the `and`, `or`, and `not` operators.
+We can evaluate boolean expressions using the `and`, `or`, and `not` operators:
```
Comment (kotp) [inline exercises/concept/ghost-gobble-arcade-game/.docs/introduction.md:12]:
```suggestion
We can evaluate Boolean expressions using the `and`, `or`, and `not` operators:
```

**Review by kotp — COMMENTED** [review]
For the "Bools and Ghouls" exercise, some suggestions…

File: concepts/bools/about.md (hunk 1 above)
Comment (BethanyG) [reply concepts/bools/about.md:5]: you are correct!  Let me get that changed...

File: concepts/bools/about.md [hunk 15]
```diff
@@ -1,21 +1,25 @@
 # About
 
-Python represents True and False values with the [bool][bool] type. There are only two boolean values: `True` and `False`. These values can be assigned to a variable and combined with the [boolean operators][boolean-operators] (`and`, `or`, `not`):
+Python represents True and False values with the [bool][bool] type.
+ There are only two boolean values in this type: `True` and `False`.
```
Comment (BethanyG) [inline concepts/bools/about.md:4]:
```suggestion
 There are only two Boolean values in this type: `True` and `False`.
```

File: concepts/bools/about.md (hunk 1 above)
Comment (BethanyG) [reply concepts/bools/about.md:5]:
```suggestion
  These values can be assigned to a variable and combined with the [Boolean operators][boolean-operators] (`and`, `or`, `not`):
```

File: concepts/bools/about.md [hunk 16]
```diff
@@ -96,25 +113,29 @@ False
...
-if flag:
-    print("Pythonistas prefer this pattern as more Pythonic.")
+>>> if flag:
+...    print("Pythonistas prefer this pattern as more Pythonic.")
 ```
 
+
 [bool-function]: https://docs.python.org/3/library/functions.html#bool
 [bool]: https://docs.python.org/3/library/stdtypes.html#truth
 [boolean-operators]: https://docs.python.org/3/library/stdtypes.html#boolean-operations-and-or-not
```
Comment (BethanyG) [inline concepts/bools/about.md:139]:
```suggestion
[Boolean-operators]: https://docs.python.org/3/library/stdtypes.html#boolean-operations-and-or-not
```

File: exercises/concept/ghost-gobble-arcade-game/.docs/hints.md (hunk 5 above)
Comment (BethanyG) [reply exercises/concept/ghost-gobble-arcade-game/.docs/hints.md:9]:
```suggestion
- You can use the [boolean][boolean] [operators][Boolean-operators] to combine arguments for a result.
```

File: exercises/concept/ghost-gobble-arcade-game/.docs/hints.md (hunk 6 above)
Comment (BethanyG) [reply exercises/concept/ghost-gobble-arcade-game/.docs/hints.md:13]:
```suggestion
- You can use the [boolean][boolean] [operator][Boolean-operators] to combine arguments for a result.
```

File: exercises/concept/ghost-gobble-arcade-game/.docs/hints.md (hunk 7 above)
Comment (BethanyG) [reply exercises/concept/ghost-gobble-arcade-game/.docs/hints.md:17]:
```suggestion
- You can use the [boolean][boolean] [operator][Boolean-operators] to combine arguments for a result.
```

File: exercises/concept/ghost-gobble-arcade-game/.docs/hints.md (hunk 8 above)
Comment (BethanyG) [reply exercises/concept/ghost-gobble-arcade-game/.docs/hints.md:21]:
```suggestion
- You can use the [boolean][boolean] [operator][Boolean-operators] to combine arguments for a result.
```

File: exercises/concept/ghost-gobble-arcade-game/.docs/hints.md [hunk 17]
```diff
@@ -2,23 +2,24 @@
...
 
 [boolean]: https://docs.python.org/3/library/stdtypes.html#truth
 [boolean-operators]: https://docs.python.org/3/library/stdtypes.html#boolean-operations-and-or-not
```
Comment (BethanyG) [inline exercises/concept/ghost-gobble-arcade-game/.docs/hints.md:24]:
```suggestion
[Boolean-operators]: https://docs.python.org/3/library/stdtypes.html#boolean-operations-and-or-not
```

**Review by kotp — APPROVED** [review]

File: exercises/concept/ghost-gobble-arcade-game/.docs/hints.md [hunk 18]
```diff
@@ -2,23 +2,24 @@
...
 
 ## 3. Define if Pac-Man loses
 
-- You can use the [boolean][boolean] [operator][boolean-operators] to combine the arguments for a result.
+- You can use the [boolean][Boolean] [operator][boolean-operators] to combine arguments for a result.
 
 ## 4. Define if Pac-Man wins
 
-- You can use the [boolean][boolean] [operator][boolean-operators] to combine the arguments for a result.
+- You can use the [Boolean][boolean] [operator][boolean-operators] to combine arguments for a result.
```
Comment (BethanyG) [inline exercises/concept/ghost-gobble-arcade-game/.docs/hints.md:21]:
```suggestion
- You can use the [Boolean][boolean] [operator][Boolean-operators] to combine arguments for a result.
```

File: exercises/concept/ghost-gobble-arcade-game/.docs/hints.md [hunk 19]
```diff
@@ -2,23 +2,24 @@
...
 
 ## 2. Define if Pac-Man scores
 
-- You can use the [boolean][boolean] [operator][boolean-operators] to combine the arguments for a result.
+- You can use the [Boolean][boolean] [operator][boolean-operators] to combine arguments for a result.
 
 ## 3. Define if Pac-Man loses
 
-- You can use the [boolean][boolean] [operator][boolean-operators] to combine the arguments for a result.
+- You can use the [boolean][Boolean] [operator][boolean-operators] to combine arguments for a result.
```
Comment (BethanyG) [inline exercises/concept/ghost-gobble-arcade-game/.docs/hints.md:17]:
```suggestion
- You can use the [boolean][Boolean] [operator][boolean-operators] to combine arguments for a result.
```

**Comment by BethanyG** [comment]
Many many thanks @kotp!  🌟

File: concepts/bools/about.md [hunk 20]
```diff
@@ -27,56 +31,68 @@ True
...
 
 >>>bool({"Pig" : 1, "Cow": 3})
 True
+
 >>>bool({})
 False
 ```
 
-When a object is used in a _boolean context_, it is evaluated transparently as _truthy_ or _falsey_ using `bool()`:
+When an object is used in a _boolean context_, it is evaluated transparently as _truthy_ or _falsey_ using `bool()`:
```
Comment (kotp) [inline concepts/bools/about.md:70]: This is one missed "Boolean" capitalization @BethanyG .

Comment (BethanyG) [reply concepts/bools/about.md:70]: Thank you for catching that!  Looks like I can't slide that one into this..so I'll do a tiny little PR-let here in a few minutes.
